### PR TITLE
2019-04-11 - What-are-Data-Lakes-and-Why-Should-You-Care

### PR DESCRIPTION
**Pull request by ahornbeck**
In a comment on the pull request, include the following information and check off the list.

- Mini bio
Ash Hornbeck is a Sr. Product Architect on the Rackspace Managed Public Clouds Product Architecture Team. With over 20 years in IT, an MBA, and a BS in Computer Engineering, Ash’s career has included roles ranging from developer to entrepreneur and everything in between. Over the past several years, he has been focused on helping large enterprises make their journey to the cloud successful.

 - Desired date of publishing
2019-04-11

 - Social media contact info
https://www.linkedin.com/in/hornbeck/

 - Redaction: (yes|no)
Yes

 - Comments: (yes|no)
No

@catlook Please review.

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by catlook** [comment]
reviewing and editing.  looks like a couple of your categories are underutilized...let me see what I can do about that!

File: _posts/2019-04-11-What-are-Data-Lakes-and-Why-Should-You-Care.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+* How will you handle data privacy (e.g., GDPR), data sovereignty, and control access so that you are not in the headlines for another data breach? _Identity and access management must be implemented to apply appropriate permissions and access controls._
+
+What about deduplication or matching records that do not share come identifying key? Is _123 Main St, Some Town, USA_ the same location as _123 Main Street, Some Town, USA_? What about _Robert Smith_ and _Bob Smith_? As you can imagine, these are simple scenarios. Much more complex situations exist in real data sets.
+
+Once the data is in the data lake and has the proper indexing and controls in place, it will need to be converted into a structured format before it can be used. Typically, this means applying the just-in-time schema as it is read. The various raw source formats are projected into a known structure so they can be consumed by ML, warehousing, or Business Intelligence (BI) systems.  (I prefer to call this process EtS-TL, Extract transform Store – Transform and Load, because some lossless data transformation is performed as the data enters the lake.  But discussion of nomenclature is a topic for another time.)
```
Comment (catlook) [inline _posts/2019-04-11-What-are-Data-Lakes-and-Why-Should-You-Care.md]: ... in place, it needs to be  ...

File: _posts/2019-04-11-What-are-Data-Lakes-and-Why-Should-You-Care.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+Once all these steps are completed, we are at a point our data analytics tools, BI systems, or ML models can begin their work. However, by leveraging a data lake we have not inadvertently destroyed potential future value found in the data. Future business questions can be explored without requiring us to know the future.
```
Comment (catlook) [inline _posts/2019-04-11-What-are-Data-Lakes-and-Why-Should-You-Care.md]: ... a point where our data ...
... data lake, we have ...

File: _posts/2019-04-11-What-are-Data-Lakes-and-Why-Should-You-Care.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+bio: "Ash Hornbeck is a Sr. Product Architect on the Rackspace Managed Public Clouds Product Architecture Team."
+categories:
+    - AWS
+    - Database
+metaTitle: "What are Data Lakes and Why Should You Care?"
+metaDescription: "A look at Data Lakes as a source of future business value."
+ogTitle: "What are Data Lakes and Why Should You Care?"
+ogDescription: "A look at Data Lakes as a source of future business value."
+---
+In recent years, Data Lakes have moved from the technology boondocks to the prime beachfront real estate of the data sciences. Why is this happening and why are they important? The short answer... there's value in there.
```
[the remark this answers is not in the document]
Comment (ahornbeck) [reply _posts/2019-04-11-What-are-Data-Lakes-and-Why-Should-You-Care.md]: done

File: _posts/2019-04-11-What-are-Data-Lakes-and-Why-Should-You-Care.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+<!-- more -->
+
+### Data Has Value
+
+Increasingly, it is apparent a company's data contains huge potential value. For many of the Internet giants like Google, Facebook, and others, their value is largely derived from their data.  But you don't have to be an Internet giant to get value from your data. Data is being used by industrial companies to predict machine failure, by financial institutions to better manage risk, by online retailers to increase customer retention, and by countless other institutions to discover new insights. It seems like a fairly simple formula: feed data to business analytics tools or machine learning (ML) systems and gain insights. In reality, it is not that simple; we must satisfy one essential prerequisite before we can tackle the analysis problem - we must actually have the data that contains the insights.
```
[the remark this answers is not in the document]
Comment (ahornbeck) [reply _posts/2019-04-11-What-are-Data-Lakes-and-Why-Should-You-Care.md]: done

File: _posts/2019-04-11-What-are-Data-Lakes-and-Why-Should-You-Care.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+### Enter Data Lakes
+
+In order to retain as much potential value as possible, we cannot discard the original data. Why don't we just keep all the data in its raw format and decide how to use it later? We could just let the data flow like rivers into a central repository, forming a "lake" of "data." :) Put another way, extract and store the data, then transform and load it as needed (often called Extract, Load, and Transform, ELT – transposing the 'T' and the 'L' in ETL).
+
+Historically, the cost of storage and extracting meaningful structure at some future date far outweighed the potential value of the information it may contain. However, the cost of storage and cost of distilling useful structure out of the chaos has dropped, making it increasingly economical for more and more companies to do just that.
+
+#### Any Collection of Raw Data is a Data Lake, Right?
+
+Ok, simple enough, just dump all of the raw data into a single location and we are done, right?  Far from it! Data ingestion must be done properly to protect customers, companies, and the data while making the it available to future data scientists and analysts. To do this, many important factors must be considered.
```
[the remark this answers is not in the document]
Comment (ahornbeck) [reply _posts/2019-04-11-What-are-Data-Lakes-and-Why-Should-You-Care.md]: done

File: _posts/2019-04-11-What-are-Data-Lakes-and-Why-Should-You-Care.md (hunk 1 above)
Comment (ahornbeck) [reply _posts/2019-04-11-What-are-Data-Lakes-and-Why-Should-You-Care.md]: done

File: _posts/2019-04-11-What-are-Data-Lakes-and-Why-Should-You-Care.md (hunk 2 above)
Comment (ahornbeck) [reply _posts/2019-04-11-What-are-Data-Lakes-and-Why-Should-You-Care.md]: done

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-1fd1d45704/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-e6eb14e359/). :bow:

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-a867d061b3/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-4475aa5cd1/)

**Pull request merged**